app/views/clients/_client_list.haml

```diff
@@ -68,9 +70,19 @@
                 %nobr
                   = masked_ssn client.ssn
                   = client_data_quality client, :ssn
-              %th.text-center
+              %td.text-center
                 = checkmark client.veteran
-              %th.text-center
+              %td.text-center
                 = checkmark client.chronic_homeless
+              %td
+                - if client.project_client.data_source_id == non_hmis_source_id
```

Comment:
This should be moved to method on project client `source_non_hmis?` (or something similar).  Both this and `is_deidentified?` can be added to client as well and delegated to the project_client.

Reply:
I made the change as requested.  I had originally done it this way so that the non_hmis_source_id would only need to be evaluated once and not in every iteration of the loop.